app/controllers/reports/regions_controller.rb

```diff
@@ -33,20 +33,19 @@ def show
         hsh[facility.name] = Reports::RegionService.new(region: facility,
                                                         period: @period).call
       }
-    else
-      @show_current_period = false
-      @dashboard_analytics = @region.dashboard_analytics(period: :month,
-                                                         prev_periods: 6,
-                                                         include_current_period: false)
     end
   end
 
   def details
     authorize { current_admin.accessible_facilities(:view_reports).any? }
 
-    @dashboard_analytics = @region.dashboard_analytics(period: @period.type, prev_periods: 6)
+    @show_current_period = true
+    @dashboard_analytics = @region.dashboard_analytics(period: @period.type,
+                                                       prev_periods: 6,
+                                                       include_current_period: true)
 
     if @region.is_a?(Facility)
+      @facility_progress = UserAnalyticsPresenter.new(@region)
```

Comment:
@claudiovallejo I think we can remove this now in favor of grabbing the data straight from `@dashboard_analytics`. AKA, replace the top "Facility Progress" table with data from `@dashboard_analytics`.